doc/CHANGELOG.md

```diff
@@ -15,6 +15,8 @@ Syntax: `- short text describing the change _(Your Name)_`
 - _()_
 - _()_
 - _()_
+- Add toggleable plant labels _(Moritz)_
+- Added a cancel confirmation to the editor tour _(Thorben)_
```

Comment:
Something went wrong with the rebase. (This is also shown like this with `git diff` so it is not a bug of GitHub.)